test/test_examples_2d_p4est.jl

```diff
@@ -60,7 +60,7 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
   @trixi_testset "elixir_euler_free_stream_p4est.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_free_stream_p4est.jl"),
       l2   = [2.063350241405049e-15, 1.8571016296925367e-14, 3.1769447886391905e-14, 1.4104095258528071e-14],
-      linf = [1.9539925233402755e-14, 2.9791447087035294e-13, 4.636291350834654e-13, 4.956035581926699e-13],
+      linf = [1.9539925233402755e-14, 1.2036205365717478e-12, 3.3154035072868737e-12, 3.3306690738754696e-12],
```

Comment:
Why did the errors grow even though you added more cells (as far as I can tell)? If you use a convergence test with this non-conforming mesh, do you get the expected EOC?

Reply:
AFAIK, the FSP errors don't change in any predictable way with the resolution.

When doing a convergence test with this mesh, I lose one order for some reason.
But this is a quite ugly mesh. It's a very strong FSP test, but I wouldn't do any other simulations on it.
The minimum scaled Jacobian is 0.036 here (or 0.076 on the refined one), which would be considered "problematic" by @fhindenlang.

Reply:
OK, sounds reasonable.